terraform/panther_gcs_transport_type_infra/production.tfvars

```diff
@@ -1,10 +1,18 @@
-project_id                           = "project-id"
-bucket_name                          = "bucket-name"
-topic_name                           = "topic-name"
-subscription_id                      = "subscription-id"
-gcp_region                           = "gcp-region"
-gcp_zone                             = "gcp-zone"
-gcs_bucket_location                  = "gcs-bucket-location"
-gcs_bucket_prefixes                  = ["logs/", "audit-logs"]
+# Basic Info
+project_id = "project-id" # ID (name) of your GCP project
+gcp_region = "gcp-region" # GCP Region that your infrastructure will live
+gcp_zone   = "gcp-zone"   # GCP Availability Zone that your infrastructure will live
+
+# Panther will pull data from this GCS bucket by way of Pub/Sub Notifications
+# If your bucket already exists you would need to remove the bucket_name and the accompanying code in the TF module
```

Comment:
`and the accompanying code`

Is it possible to be more specific here you think? E.g. could we point them to a specific line and have a comment in that line(s) that needs to be removed?

Reply:
Reworded. Let me know what you think.